services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/PersistenceOperationsBolt.java

```diff
@@ -66,6 +67,11 @@ protected void handleInput(Tuple input) throws AbstractException {
                     e.getMessage(),
                     "ISL does not exist.");
             getOutput().emit(StreamType.ERROR.toString(), input, new Values(data, correlationId));
+        } catch (SwitchNotFoundException e) {
```

Comment:
Something wrong with this. Why generic PersistenceOperationsBolt should care about specific SwitchNotFoundException?